downloaded/lisk_bridge.sh

```diff
@@ -20,20 +20,24 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 ######################################################################
 
+# shellcheck disable=SC1090
+. "$(pwd)/shared.sh"
+# shellcheck disable=SC1090
+. "$(pwd)/env.sh"
+
 # Declare working variables
 TARGET_HEIGHT="3513100"
 BRIDGE_HOME="$(pwd)"
 BRIDGE_NETWORK="main"
 LISK_HOME="$HOME/lisk-main"
-JQ="$LISK_HOME/bin/jq"
```

Comment:
Is `jq` a dependency now?

Reply:
it sources env.sh, so it should still be able to use the bundled jq